contrib/go/src/python/pants/contrib/go/register.py

```diff
@@ -45,5 +46,6 @@ def register_goals():
   task(name='go', action=GoCompile).install('compile')
   task(name='go', action=GoBinaryCreate).install('binary')
   task(name='go', action=GoRun).install('run')
-  task(name='gofmt', action=GoFmt).install('compile')
+  task(name='go-checkstyle', action=GoCheckstyle).install('compile')
```

Comment:
This is technically a breaking change... can you add a deprecated scope for this task?